templates/core/core-dpl.yaml

```
@@ -4,18 +4,18 @@ metadata:
   name: {{ template "harbor.core" . }}
   labels:
 {{ include "harbor.labels" . | indent 4 }}
-    component: core
+{{ include "harbor.labels" . | indent 4 }}
```

duplicated?

---

Oops, typo. It had to be:
```suggestion
{{ include "harbor.core.labels" . | indent 4 }}
```

---

Fixed it.